src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -41,11 +41,12 @@
     <DocumentationFile>
     </DocumentationFile>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(OS)'!='Unix'">
+    <EnableNode>True</EnableNode>
+    <EnablePowerShell>True</EnablePowerShell>
+    <DefineConstants>$(DefineConstants);WINDOWS;FEATURE_NODE;FEATURE_POWERSHELL</DefineConstants>
+  </PropertyGroup>
   <ItemGroup>
-    <Reference Include="EdgeJs, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Rather than completely restructure/reorder these project files into item groups, don't the Reference/Compile elements support a Condition attribute so this can simply be tacked on as needed? This will make this much easier to review and maintain. E.g. in this review I'd then be able to see that you're simply adding Conditions to existing elements - with the reordering I honestly can't tell if you've accidentally added/removed things, etc.